noah/src/main/scala/edu/uci/ics/cloudberry/noah/crawler/JarRunner.java

```diff
@@ -0,0 +1,164 @@
+package edu.uci.ics.cloudberry.noah.crawler;
+
+
+import com.crawljax.browser.EmbeddedBrowser.BrowserType;
+
+import com.crawljax.condition.UrlCondition;
+import com.crawljax.core.CrawljaxException;
+import com.crawljax.core.CrawljaxRunner;
+import com.crawljax.core.configuration.BrowserConfiguration;
+import com.crawljax.core.configuration.CrawljaxConfiguration;
+import com.crawljax.core.configuration.CrawljaxConfiguration.CrawljaxConfigurationBuilder;
+import com.crawljax.plugins.crawloverview.CrawlOverview;
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.base.Charsets;
+import com.google.common.io.Resources;
+import org.apache.commons.cli.ParseException;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+
+public class JarRunner {
```

Comment:
Yes... Only dat part is left... To remove JarRunner... But I still need to
analyze JarRunner a bit and make sure I don't miss anything.... If that
part is done we can just make CrawlerDriver do the required job... :)
On Jun 22, 2016 11:14 AM, "Jianfeng Jia" <notifications@github.com> wrote:

In noah/src/main/scala/edu/uci/ics/cloudberry/noah/crawler/JarRunner.java
<https://github.com/ISG-ICS/cloudberry/pull/100#discussion_r68104469>:

> +import com.crawljax.core.CrawljaxException;
> +import com.crawljax.core.CrawljaxRunner;
> +import com.crawljax.core.configuration.BrowserConfiguration;
> +import com.crawljax.core.configuration.CrawljaxConfiguration;
> +import com.crawljax.core.configuration.CrawljaxConfiguration.CrawljaxConfigurationBuilder;
> +import com.crawljax.plugins.crawloverview.CrawlOverview;
> +import com.google.common.annotations.VisibleForTesting;
> +import com.google.common.base.Charsets;
> +import com.google.common.io.Resources;
> +import org.apache.commons.cli.ParseException;
> +
> +import java.io.File;
> +import java.io.IOException;
> +import java.util.concurrent.TimeUnit;
> +
> +public class JarRunner {

I don't think it is JarRunner. Could we just use CrawlerDriver?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/ISG-ICS/cloudberry/pull/100/files/b541c7e50305cce36a85e40855e7ece1ef680670#r68104469>,
or mute the thread
<https://github.com/notifications/unsubscribe/ARXIuzOAMaWrBbwf6UaCdPkgeLOqyqRyks5qOXnkgaJpZM4I8BJd>
.